pkg/madmin/API.md

```diff
@@ -326,6 +328,64 @@ __Example__
     log.Println("SetConfig: ", string(buf.Bytes()))
 ```
 
+<a name="GetConfigKeys"></a>
+### GetConfigKeys(keys []string) ([]byte, error)
+Get a json document which contains a set of keys and their values from config.json.
+
+__Example__
+
+``` go
+    configBytes, err := madmClnt.GetConfigKeys([]string{"version", "notify.amqp.1"})
+    if err != nil {
+        log.Fatalf("failed due to: %v", err)
+    }
+
+    // Pretty-print config received as json.
+    var buf bytes.Buffer
+    err = json.Indent(buf, configBytes, "", "\t")
+    if err != nil {
+        log.Fatalf("failed due to: %v", err)
+    }
+
+    log.Println("config received successfully: ", string(buf.Bytes()))
+```
+
+
+<a name="SetConfigKeys"></a>
+### SetConfigKeys(params map[string]string) (SetConfigResult, error)
+Set a set of keys and values to the Minio server or distributed setup and restart for configuration
```

Comment:
`Set a set of keys and values to the Minio server or distributed setup and restart for configuration change to take effect.` 
 =>
`Set a set of keys and values`**`for`**`Minio server setup and restart`**`the Minio Server`**`for the new configuration change`**`s`**`to take effect.`